### ISSUE TITLE
Only the top 100 shards are shown

### ISSUE THREAD

**Issue by RX14** [zamith/crystalshards#6]

**Comment by zamith** [comment]
@RX14 Yes, I know. To show more I'd have to make further calls to github until there are no more results. At first this was not needed, because there weren't that many shards, now I just felt that since you can filter and order, it might not even be that useful.

Did you feel the need to see more? How did you notice this?

**Comment by RX14** [comment]
Well I was looking for my library and I couldn't find it, having a list is rather pointless if it isn't exhaustive.

**Comment by zamith** [comment]
I'd welcome a PR. :)
